client/src/components/TopBar.js

```diff
@@ -59,21 +55,22 @@ const TopBar = ({
   useEffect(() => {
     let output = false
     if (
-      visibilitySetting === visible.USERS &&
+      visibilitySetting === GENERAL_BANNER_VISIBILITIES.USERS_ONLY.value &&
       currentUser &&
       !currentUser.isSuperUser()
     ) {
       output = true
     }
     if (
-      visibilitySetting === visible.SUPER_USERS &&
+      visibilitySetting ===
+        GENERAL_BANNER_VISIBILITIES.SUPER_USERS_AND_ADMINSTRATORS.value &&
       currentUser &&
       currentUser.isSuperUser()
     ) {
       output = true
     }
     if (
-      visibilitySetting === visible.USERS_AND_SUPER_USERS &&
+      visibilitySetting === GENERAL_BANNER_VISIBILITIES.ALL.value &&
       (currentUser || currentUser.isSuperUser())
     ) {
       output = true
```

Comment:
This could be merged into a single expression: `const output = currentUser && ((visibilitySetting === GENERAL_BANNER_VISIBILITIES.USERS_ONLY.value && !currentUser.isSuperUser()) || (visibilitySetting ===         GENERAL_BANNER_VISIBILITIES.SUPER_USERS_AND_ADMINSTRATORS.value && currentUser.isSuperUser()) || (visibilitySetting === GENERAL_BANNER_VISIBILITIES.ALL.value))`